plugins/node/opentelemetry-instrumentation-dataloader/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@opentelemetry/instrumentation-dataloader",
+  "version": "0.30.0",
+  "description": "OpenTelemetry instrumentation for dataloader",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "clean": "rimraf build/*",
+    "compile": "tsc -p .",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version && lerna run version:update --scope @opentelemetry/instrumentation-dataloader --include-dependencies",
+    "prewatch": "npm run precompile",
+    "prepare": "npm run compile",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "test-all-versions": "tav",
+    "version:update": "node ../../../scripts/version-update.js"
+  },
+  "keywords": [
+    "dataloader",
+    "instrumentation",
+    "nodejs",
+    "opentelemetry",
+    "profiling",
+    "tracing"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.12.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "peerDependencies": {
+    "@opentelemetry/api": "^1.0.0"
+  },
+  "devDependencies": {
+    "@opentelemetry/api": "^1.0.0",
+    "@opentelemetry/context-async-hooks": "^1.3.1",
```

Comment:
Is there a reason for using 1.3.1 which is not the newest but also not the oldes supported?

Reply:
TBH, I just copy-pasted some other instrumentation as the starting point, will bump these up.